hierarc/Likelihood/LensLikelihood/ddt_hist_likelihood.py

```diff
@@ -25,25 +26,36 @@ def __init__(self, z_lens, z_source, ddt_samples, kde_kernel=None, ddt_weights=N
         :param kde_kernel: string of KDE kernel type
         :param bandwidth: bandwith of kernel
         :param nbins_hist: number of bins in the histogram
-        :param normalized: bool, if True, returns the normalized likelihood, if False, separates the constant prefactor
-         (in case of a Gaussian 1/(sigma sqrt(2 pi)) ) to compute the reduced chi2 statistics
+        :param normalized: bool, if True, returns the normalized likelihood, 
+        if False, separates the constant prefactor (in case of a Gaussian 
```

Comment:
formatting: If the documentation goes to the second line, please insert by one spacing to be conformal with the sphinx documentation scheme

Reply:
This only applies for parameter docstrings (that start with colons), correct? If so, done!